Basic-Car-Maintenance/Shared/Dashboard/ViewModels/DashboardViewModel.swift

```diff
@@ -67,7 +67,8 @@ class DashboardViewModel {
     func getMaintenanceEvents() async {
         if let uid = authenticationViewModel.user?.uid {
             let db = Firestore.firestore()
-            let docRef = db.collection("maintenance_events").whereField("userID", isEqualTo: uid)
+            let docRef = db.collection(FirestoreCollection.maintenanceEvents)
+                .whereField(FirestoreCollection.userID, isEqualTo: uid)
```

Comment:
Can you actually also add a new enum called `FirestoreField` for the `userID`, because it's not technically a "collection"

Reply:
Will do! Would you rather the file name be something that gives a better clue? 

Instead of just "Constants" maybe "FirestoreConstants" 

Thoughts?